docs/Version-Comparison.md

```diff
@@ -54,6 +54,8 @@ When in doubt, new users are advised to try out OTP2 and switch to OTP1 if they
 | Transfer Priority | yes (route, trip, stop level) | no (planned) |
 / REST API format | XML, JSON | JSON only |
 
+[1] [Trip Frequency is not working in OTP 2.0.](https://github.com/opentripplanner/OpenTripPlanner/issues/3243)
```

Comment:
Frequencies.txt is a standard part of GTFS so if it is not supported in OTP2 this probably deserves to be more than a footnote - although GTFS for big networks often uses purely scheduled trips, some people working with other GTFS inputs may be confused or waste time if their frequency trips are not imported.